code/game/objects/structures/girders.dm

```diff
@@ -190,6 +190,10 @@
 				return
 
 		if(!sheets.has_unique_girder && sheets.material_type)
+			if(istype(src, /obj/structure/girder/reinforced))
+				to_chat(user, span_warning("You need plasteel to make a reinforced wall!"))
```

Comment:
```suggestion
				user.balloon_alert("you need plasteel!")
```

Reply:
If you want this, it's better to do it in another PR for the 62 `to_chat` messages that need to be replaced in this file. This makes more sense than _ballooning_ this 1-line hotfix PR.  I can do this after this PR gets merged so there aren't conflicts.